src/API/index.js

```diff
@@ -33,3 +33,8 @@ export const fetchImageClones = async (composeID) => {
   const { data } = await axios.get(imageBuilderURL(`composes/${composeID}/clones`));
   return data;
 };
+
+export const fetchReservationByProvider = async (reservationID, provider) => {
```

Comment:
Why is this here when there are no backend changes? :thinking:

Reply:
this is for fetching the specfic aws reservation, with the instances list.